README.md

```diff
@@ -41,23 +39,22 @@ python myScript
 deactivate
 ```
 
-
-## Grammar Correction: Technical Details
+# Grammar Correction: Technical Details
 
 This repository contains the scripts for creating synthetic data and training a grammar model with spaCy.
 
 ### Grammar
 
 Quill has developed a grammar pipeline that labels sentences with frequent grammar errors, such as subject-verb agreement errors and plural-possessive errors.
 The goal is to give students feedback on their writing, so that they can correct grammatical errors.
-This pipeline is a combination of simple rules and a machine-learning model. The machine-learning model is trained on a mix of real data from students and data with
+This pipeline is a combination of simple rules and a statistical machine learning model. The machine learning model is trained on a mix of real data from students and data with
```

Comment:
What about combining the first two sentences:
"This pipeline is a combination of simple rules and a statistical machine learning modeling model that is trained on a mix real student data along with synthetic data generated from grammar errors."